src/backend/legendary/library.ts

```diff
@@ -1,3 +1,4 @@
+import { LegendaryInstallPlatform } from './../../common/types/legendary'
```

Comment:
```suggestion
import { LegendaryInstallPlatform } from 'common/types/legendary'
```
You might want to change `JavaScript -> Preferences: Import Module Specifier` to "shortest" (or "non-relative") in your VSCode settings

Reply:
weird, I already had that in `shortest`, I'll check again next time it auto imports something

Reply:
I also remember having issues with this back when I was using VSCode. We probably have to explicitly tell it about those import paths *somewhere*